sceptre/plan/actions.py

```diff
@@ -1000,3 +1000,147 @@ def diff(self, stack_differ):
         :rtype: sceptre.diffing.stack_differ.StackDiff
         """
         return stack_differ.diff(self)
+
+    @add_stack_hooks
+    def drift_detect(self) -> Tuple[str, dict]:
+        """
+        Show stack drift for a running stack.
+
+        DETECTION_COMPLETE and DETECTION_FAILED
+        are 2 of the 3 DetectionStatuses from the
+        DescribeStackDriftDetectionStatus API. TIMED_OUT is
+        something we have defined in the event that the wait
+        continues to return DETECTION_IN_PROGRESS after our
+        timeout.
+
+        :returns: The status and resource drifts.
+        """
+        try:
+            status = self._get_status()
+        except StackDoesNotExistError:
+            self.logger.info(f"{self.stack.name} - Does not exist.")
+            status = StackStatus.COMPLETE
+            return (status, {})
+
+        response = self._detect_stack_drift()
+
+        detection_id = response["StackDriftDetectionId"]
+        detection_status, stack_drift_status = self._wait_for_drift_status(detection_id)
+
+        return (detection_status, stack_drift_status)
```

Comment:
Rather than returning a tuple here, I think it should just return the drift status response, since there is [other information](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudformation.html#CloudFormation.Client.describe_stack_drift_detection_status) that callers of this method might want, such as the drifted resource count, detection status reason, etc...